static/js/components/ProfileImage.js

```diff
@@ -7,18 +7,39 @@ import type { Profile } from '../flow/profileTypes';
 
 export default class ProfileImage extends React.Component {
   props: {
-    profile: Profile
+    profile: Profile,
+    editable: boolean
+  };
+
+  static defaultProps = {
+    editable: false
   };
 
   render () {
-    const { profile } = this.props;
+    const { profile, editable } = this.props;
     const imageUrl = makeProfileImageUrl(profile);
+    let cameraIcon = () => {
+      if (editable) {
+        return (
+          <span className="img">
+            <img
+              src='/static/images/camera.png'
+              alt="Camera image for profile image"
+            />
+          </span>
+        );
+      }
+    };
```

Comment:
with material-design the above function would look something like:

```js
const cameraIcon = editable => editable ? <Icon name="camera" /> : null;
```

assuming the name of the appropriate icon is `camera`.